visualization/conf/customWebpack.config.js

```diff
@@ -0,0 +1,27 @@
+const RemarkHTML = require("remark-html")
+
+module.exports = config => {
+	config.module.rules.push(
+		{
+			test: /\.glsl$/,
+			use: ["webpack-glsl-loader"]
+		},
+		{
+			test: /\.md$/,
+			use: [
+				{ loader: "html-loader" },
+				{
+					loader: "remark-loader",
+					options: {
+						removeFrontMatter: false,
+						remarkOptions: {
+							plugins: [RemarkHTML]
+						}
+					}
+				}
+			]
+		}
+	)
+
+	return config
+}
```

Comment:
Maybe rename the file to `webpack.config.js`? All webpack config files should be "custom" in a way, no?

Reply:
done